src/v2/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -407,8 +401,7 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
     }
 
     const shouldDisplayMakeOfferButton: boolean | null =
-      isOfferable ||
-      (isPriceListed && isOfferableFromInquiry && makeOfferEnabled)
+      isOfferable || (isPriceListed && isOfferableFromInquiry)
```

Comment:
We shouldn't have to check for `(isPriceListed && isOfferableFromInquiry)` anymore. `ForSale#offerable?` is handling these:
https://github.com/artsy/gravity/blob/e69d1154a6371249d791ede0d765c15ef8671473/app/models/util/for_sale.rb#L290-L294
https://github.com/artsy/gravity/blob/e69d1154a6371249d791ede0d765c15ef8671473/app/models/util/for_sale.rb#L383-L384

There wouldn't be much problem about being redundant, but the issue here is that this check is not taking partner `inquiry_order_enabled` flag into account, which means that we display the `Make Offer` button for any artwork that's offerable from inquiry in case the price is not hidden (even if `inquiry_order_enabled` is off).

I would suggest removing this entirely.
If we want to be extra careful for some reason and we don't want to remove it yet, I guess it would be better to also add the inquiry_order_enabled check then.

Reply:
Makes sense. Removing the other two now, and will ticket a cleanup for this file which is a bit off, but it's out of this scope.